README.md

```diff
@@ -1,24 +1,80 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Budget-App
 
-Things you may want to cover:
+> This web application is a Ruby on Rails application that allows you to manage your budget in a professional way. You can register and/or log in and your data will be safely saved and you will be the only one person to manage access them. You can perform transaction transactions associated with a category of your transaction then at the end of the day you can see the money spent on each category.
 
-* Ruby version
 
-* System dependencies
+## Built With
 
-* Configuration
+- Major: Ruby, HTML,CSS
+- Frameworks: Rails, Bootstrap5
+- Libraries: Cancancan, Devise
+- Technologies: Git, Gitflow, Linters, Github, PostgreSql
 
-* Database creation
+## Live Demo (not available yet)
 
-* Database initialization
+[Live Demo Link](https://budgetify-ap.herokuapp.com/)
```

Comment:
* Your live version link doesn't work as when i try to signup it throws an error , Kindly fix this error 
![image](https://user-images.githubusercontent.com/52497246/198209755-7b169732-1902-4eab-a7ca-ba0b21970f6e.png)